src/pudl/etl/__init__.py

```diff
@@ -16,7 +16,6 @@
     load_asset_checks_from_modules,
     load_assets_from_modules,
 )
-from dagster._core.definitions.cacheable_assets import CacheableAssetsDefinition
```

Comment:
We're importing this class from a private module and not using it, and where it is within the Dagster library has changed, so I'm removing it for now.